apps/web/next.config.js

```diff
@@ -7,7 +7,7 @@ const nextConfig = {
       {
         source: '/',
         destination: '/alfajores',
-        permanent: true,
+        permanent: false,
```

Comment:
Not sure what this does

Reply:
301 vs 302 redirects: with a permanent redirect the browser stores the redirect locally and so doesnt ask server again just goes to new location. this is annoying in local dev as localhost:3000 always redirects even for other servers and even worse when we remove alfajores (gonna need to do a double redirect in some cases)